configure-replication-zones.md

```diff
@@ -42,7 +42,7 @@ Field | Description
 ------|------------
 `range_min_bytes` | Not yet implemented.
 `range_max_bytes` | The maximum size, in bytes, for a range of data in the zone. When a range surpasses this size, CockroachDB will spit it into two ranges.<br><br>**Default:** `67108864` (64MiB)
-`ttlseconds` | The number of seconds overwritten values will be retained before garbage collection. Smaller values can save disk space if values are frequently overwritten; larger values increase the range allowed for `AS OF SYSTEM TIME` queries. It is not recommended to set this below `600` (10 minutes).<br><br>**Default:** `86400` (24 hours)
+`ttlseconds` | The number of seconds overwritten values will be retained before garbage collection. Smaller values can save disk space if values are frequently overwritten; larger values increase the range allowed for `AS OF SYSTEM TIME` queries, also know as [Time Travel Queries](select.html#select-historical-data-time-travel).<br><br>It is not recommended to set this below `600` (10 minutes); doing so will cause problems for long-running queries. Also, in the case of rapidly-changing data, it is not recommended to set this above the default; doing so can lead to memory spikes on the server because all versions of a row are stored together.<br><br>**Default:** `86400` (24 hours)
```

Comment:
I think it's worth including the quantitative rule of thumb here instead of just the vague "rapidly changing". (in some cases rapidly-changing data could require reducing `ttlseconds` from its default).